.travis.yml

```diff
@@ -25,8 +25,6 @@ before_install:
 script:
   - sh ci.sh
 
-after_failure: "cat /home/travis/build/zetaapps/utils/zetautils/build/outputs/lint-results-debug.html"
```

Comment:
This just made the log file appear in the travis web interface, which was useful at the time.  I'm guessing there is a way to pull the log files off the VM directly, but I didn't dive into it.